lib/Toast.js

```diff
@@ -32,16 +32,16 @@ class Toast extends Component {
 
     _toast = null;
 
-    componentWillMount = () => {
+    componentDidMount = () => {
         this._toast = new RootSiblings(<ToastContainer
             {...this.props}
             duration={0}
         />);
     };
 
-    componentWillReceiveProps = nextProps => {
+    componentDidUpdate = prevProps => {
```

Comment:
To prevent re-rendering it would be a good idea to add some check to see if `this.props` and `prevProps` is not equal to each other